### ISSUE TITLE
Umlauts being in wrong position in font?

### ISSUE THREAD

**Issue by machosallad** [sschueller/vbz-fahrgastinformation#3]
Hi Stefan, I'm in the works of creating a very similar project as you have made. Since I'm from Sweden there are a few non-English letters I would like to use (same situation for you). Struggling a bit with the font creation and while I was looking through your code I noticed your comment about replacing letters with the wrong position in the font (see below). I'm a bit curious, what's the reason behind it?

Thanks in advance!

```
String Display::cropDestination(String destination)
{
    // remove extra text
    destination.replace("Zürich,", "");
    destination.replace("Winterthur,", "");
    destination.replace("Bahnhof ", "");

    // fix umlauts being in wrong position in font
    destination.replace("ä", "\x7B");
    destination.replace("ö", "\x7C");
    destination.replace("ü", "\x7D");
```

**Comment by sschueller** [comment]
Hi,

I think the issue I had was that I placed the umlauts in the first 128 character space as I did not want to use the extended ASCII which would be required to do it correctly. 

I have since rewritten most of the code including the font but I can not publish it at this time as I have made a product out of it: https://stationdisplay.com/. I do plan on releasing code eventually but I think I may still be missing the characters you need?

Let me know if you have any other questions regarding the font.

**Comment by machosallad** [comment]
Thanks for the quick reply. I can fully understand you not publishing all new changes since you made a product out of it, congratulations!  Saw our article about turning an idea into a product and it was insightful to read. 

Since I posted my question I dived deeper into what Adafruit had in their documentation and one of their phrases was hard do understand. Either they meant only 7-bit characters was supported and 8-bit would be available in the future, or, they were referring to the fonts they provide as ready made .h-files. That 8-bit would be provided in the future rather than supported. This lead me into the right direction as is exactly what you answered. The limitation of 128 characters - my characters were in the 200+ range. 

So my solution reminds of yours. I simply replaced some characters that I would likely not be using in the normal case. Placed my foreign characters there and whenever I want to print something I replace any occurrences of foreign chars in the string with my alternatives.  

So I think my problem is solved, and thanks for an inspiring project!
